### DISCUSSION TITLE
Allow `await` multiple signals

### DISCUSSION THREAD

**Discussion by acgc99** [godotengine/godot-proposals#8437] [Scripting]
I'm currently working on an app. For switching between screens/pages, I would like to do something like this:
```
await old_page.finished and page.finished
```
To make sure that the old and new pages are fully loaded before starting the switching animation.

If I type that, I get the warning:
```
Line 57 (STANDALONE_EXPRESSION):Standalone expression (the line has no effect).
```
I guess that the second signal is ignored.

This could also be extended with `or` or complex conditions (just like an `if` statement).

**Comment by AThousandShips** [comment]
Thank you for your proposal, closing as a duplicate of:
* https://github.com/godotengine/godot-proposals/issues/6243
